Friend/firmware/firmware_v1.0/prj_xiao_ble_sense_devkitv1-spisd.conf

```diff
@@ -34,11 +34,11 @@ CONFIG_BT_PHY_UPDATE=y
 CONFIG_BT_BAS=y
 CONFIG_BT_DIS=y
 CONFIG_BT_DIS_PNP=n
-CONFIG_BT_DIS_MODEL="Friend"
+CONFIG_BT_DIS_MODEL="Friend DevKit 1 SPI SD"
 CONFIG_BT_DIS_MANUF="Based Hardware"
 CONFIG_BT_DIS_FW_REV=y
 CONFIG_BT_DIS_HW_REV=y
-CONFIG_BT_DIS_FW_REV_STR="1.0.5"
+CONFIG_BT_DIS_FW_REV_STR="1.0.6"
```

Comment:
We haven't produced official 1.0.5, so don't think we need to bump to 1.0.6.

Reply:
@francip rolledback that particular change